src/System.Device.Gpio/System/Device/Gpio/Drivers/SysFSDriver.cs

```diff
@@ -0,0 +1,502 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.IO;
+using System.Linq;
+using System.Threading;
+
+namespace System.Device.Gpio.Drivers
+{
+    internal class SysFSDriver : UnixDriver
+    {
+        private const string GpioBasePath = "/sys/class/gpio";
```

Comment:
I suppose the rest of this file is just a straight copy of what the UnixDriver was? If not let me know so that we can review the changes. Also, this file should have the .Linux.cs ending as we don't want to compile it on the Windows side.

Reply:
Yes it is just copy, but since i have copied seems there were some code change, so am gonna copy again from latest code 

`Also, this file should have the .Linux.cs ending as we don't want to compile it on the Windows side.`
yeah i saw build failing on this class most likely because of that, will update